utility/Format.php

```diff
@@ -7,12 +7,33 @@
  */
 class Format
 {
+    /**
+     * Convert a Julian datetime value into a human-readable datetime string.
+     *
+     * @param float $jdt Julian datetime value.
+     * @return string
+     */
+    public static function date(float $jdt): string
+    {
+        $r = swe_jdut1_to_utc($jdt, SE_GREG_CAL);
+        $sec = floor($r['sec']);
+
+        return sprintf(
+            '%s %s %s  %s:%s:%s UT',
```

Comment:
The double-space between the day and hour were in the original method, so I preserved them here.